src/Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj

```diff
@@ -27,11 +27,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="$(MSBuildThisFileDirectory)Modules\**"
-            Link="Modules\%(RecursiveDir)%(FileName)%(Extension)"
-            PackagePath="contentFiles/any/any/Modules"
-            PackageCopyToOutput="true"
-            CopyToOutputDirectory="PreserveNewest" />
+    <Content Include="$(MSBuildThisFileDirectory)Modules\**" Link="Modules\%(RecursiveDir)%(FileName)%(Extension)" PackagePath="contentFiles/any/any/Modules" PackageCopyToOutput="true" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Given the below `Content` properties are all in multi-line format, maybe you should revert this change?

Reply:
done